### PR TITLE
[examples][frameworks] add tests for all examples being detected

### PR DESCRIPTION
**Pull request by EndangeredMassa**
This PR adds tests (under `packages/fs-detectors`) that ensure are `./examples` get detected as the appropriate framework.

### CODE REVIEW

File: packages/frameworks/src/frameworks.ts [hunk 1]
```diff
@@ -1932,6 +1934,7 @@ export const frameworks = [
     ],
   },
   {
+    // TODO: fix detected as "vite"
```
Comment (EndangeredMassa) [inline packages/frameworks/src/frameworks.ts]: Should "Hydrogen" be moved above "Vite"?

PR where Hydrogen was added: https://github.com/vercel/vercel/pull/8073

File: examples/sveltekit/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   },
   "devDependencies": {
     "@sveltejs/adapter-auto": "next",
-    "@sveltejs/kit": "next",
+    "@sveltejs/kit": "next TODO ",
```
Comment (EndangeredMassa) [inline examples/sveltekit/package.json]: `sveltekit` detection (as opposed to `sveltekit-1` detection) looks for a package version that looks like this part of the regex `"1\\.0\\.0-next\\.(\\d+)"`. Should we update the `./examples/sveltekit` example to match?

This is the PR where it was changed: https://github.com/vercel/vercel/pull/9070/files#diff-837505f1cd8d9a3b7e5986eb08b96e256bf9e5d8d74ce1ac973fe0db4a4961f9L1008

Comment (EndangeredMassa) [reply examples/sveltekit/package.json]: @tcc-sejohnson @leerob what do you think?

File: packages/frameworks/src/frameworks.ts (hunk 1 above)
Comment (EndangeredMassa) [reply packages/frameworks/src/frameworks.ts]: @TooTallNate @leerob what do you think?

File: examples/sveltekit/package.json (hunk 2 above)
Comment (styfle) [reply examples/sveltekit/package.json:15]: cc @tcc-sejohnson

File: packages/frameworks/src/frameworks.ts (hunk 1 above)
Comment (styfle) [reply packages/frameworks/src/frameworks.ts]: I think the answer is yes.

Also cc @TooTallNate

Comment (TooTallNate) [reply packages/frameworks/src/frameworks.ts]: I think yes.

Comment (EndangeredMassa) [reply packages/frameworks/src/frameworks.ts]: done: [`f880f35` (#9197)](https://github.com/vercel/vercel/pull/9197/commits/f880f355022a32c845fc589a188c5399d2a63d8b)

File: examples/sveltekit/package.json (hunk 2 above)
Comment (dummdidumm) [reply examples/sveltekit/package.json]: Makes sense to do this - if in the future we do `2.0.0-next` just `next` would point to that, so this should be changed to `1.0.0-next.589` (the last prerelease version)

Comment (EndangeredMassa) [reply examples/sveltekit/package.json]: Done! [`56e2857` (#9197)](https://github.com/vercel/vercel/pull/9197/commits/56e2857bcad8eadbd768a3f04a40234405174aa5)

**Review by cb1kenobi — APPROVED** [review]
LGTM!

File: examples/sveltekit/package.json (hunk 2 above)
Comment (styfle) [reply examples/sveltekit/package.json]: Lets add a lock file if there isn't already one

**Review by TooTallNate — APPROVED** [review]

**Review by leerob — APPROVED** [review]

**Review by cb1kenobi — APPROVED** [review]

**Pull request merged**